deploying.md

```diff
@@ -0,0 +1,113 @@
+# Deploying Philanthropy Data Commons software
```

Comment:
Typically, top-level documentation files, like `README.md` and `CONTRIBUTING.md` and such, have names in all caps.  I don't know why, but that's the convention (I suppose I could guess at reasons why that convention has evolved -- I do think it's a good one).  By that convention, this should be `DEPLOYMENT.md`.